templates/deployment.yaml

```diff
@@ -59,7 +59,7 @@ spec:
             exec:
               # Stop worker from picking up more jobs and send SIGKILL once /worker/poke is no longer available (All jobs have completed)
               # Added sleep 30 at start to allow enough time for container to start up
-              command: ["/bin/sh", "-c", "sleep 30 && curl -X 'DELETE' http://localhost:5000/worker/shutdown && until [[ $(curl -I --silent -o /dev/null -w %{http_code} http://localhost:5000/worker/poke) != 200  ]] ;do printf '.'; sleep 5; done;"]
+              command: ["/bin/sh", "-c", "sleep 30 && if [ $(curl -X 'DELETE' -I --silent -o /dev/null -w %{http_code} http://localhost:5000/worker/shutdown) == 202 ] ; then until [[ $(curl -I --silent -o /dev/null -w %{http_code} http://localhost:5000/worker/poke) != 200  ]] ;do printf '.'; sleep 5; done; else kill -11 1; fi;"]
```

Comment:
Curious as to why you are using kill -11 instead of -9 or -15

Reply:
Hi Austin,

I have tried initially with SIGKILL(-9) and SIGTERM(-15), but not working as expected. Not even killing the process, specifically for PID-1. Then I tried to use SIGSEGV(-11), it's working.

Thanks
Manoj